tests/apollo/test_skvbc_chaotic_startup.py

```diff
@@ -37,7 +37,7 @@ def start_replica_cmd(builddir, replica_id, view_change_timeout_milli="10000"):
             "-i", str(replica_id),
             "-s", statusTimerMilli,
             "-v", view_change_timeout_milli,
-            "-f", '0' # disable time service until intermittent failures are fixed for this test
+            "-f", '1'
```

Comment:
As time service is true by default, all we should do is to remove -f from the command. And anytime we need to configure with disabled TS, we should add -f 0.

Reply:
done